test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperPasswordTest.cs

```diff
@@ -345,6 +345,40 @@ public static TheoryData PasswordFor_WithComplexExpressionsData
             Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result));
         }
 
+        [Fact]
+        public void PasswordHelpers_UsesSpecifiedExpression()
```

Comment:
No, `Password()` and `PasswordFor()` ignore the expression value, `ViewData`, `ModelState`, and so on. _Only_ the passed `htmlAttributes` or `value` arguments can results in a generated `value` attribute. (And in the `PasswordFor()` case, there's no `value` argument.)

Need to change this test name and test a non-`null` `Property1`